### ISSUE TITLE
Add tornado.general facility to refactored logging

### ISSUE THREAD

**Issue by fnoop** [goodrobots/maverick-api#52]
tornado.access and tornado.application handled, tornado.general isn't.

**Comment by fnoop** [comment]
Also ensure rotation.
Old code:
```
handler = RotatingFileHandler(
    "tornado.log",
    mode="a",
    maxBytes=1 * 1024 * 1024,
    backupCount=2,
    encoding=None,
    delay=0,
)
```

**Comment by SamuelDudley** [comment]
implemented in fccd52aa723529c848530abde2ab24d390173ea8
